tests/Application/.env.test

```diff
@@ -4,6 +4,6 @@ KERNEL_CLASS='Tests\BitBag\SyliusExamplePlugin\Application\Kernel'
 
 DATABASE_URL=sqlite:///%kernel.project_dir%/var/data.db
 
-MAIL_CHIMP_API_KEY=6a74ec890676264ec8116a7945929d12-us18
-
-MAIL_CHIMP_LIST_ID=8af647b5c0
+MAIL_CHIMP_API_KEY='5cf6212ae1aa1b465f5c53124b9afb54-us1'
+MAIL_CHIMP_LIST_ID='237417b17d'
+MAIL_CHIMP_WEBHOOK_SECRET='237417b17d'
```

Comment:
Should this be commited?

Reply:
Good point, Mailchimp would disable any keys that are publicly visible, I'm going to change it to dummy text